test/CMakeLists.txt

```diff
@@ -1104,7 +1122,8 @@ else()
     )
 endif() #if CODECOV_TEST
 
-add_custom_test(test_conv_igemm_dynamic SKIP_UNLESS_ALL
+if(NOT (MIOPEN_TEST_GFX1030 AND WORKAROUND_ISSUE_1054))
+add_custom_test(test_conv_igemm_dynamic SKIP_UNLESS_ALL GFX1030_ENABLED
```

Comment:
@atamazov this test `test_conv_igemm_dynamic` is only for vega. and here should be `GFX1030_DISABLED`. In this case the `WORKAROUND_ISSUE_1054` can be omitted

Reply:
Fixed, thanks.

Reply:
@atamazov should be `GFX1030_DISABLED` or just omit?

Reply:
@atamazov should be `GFX1030_DISABLED` or just ommited?

Reply:
@carlushuang Omitting is enough, see https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1038/files#r677834654

Reply:
@carlushuang Now I see)) Previous commit was incorrect. Fixed.

Reply:
@atamazov ok